src/CalcManager/CEngine/Number.cpp

```diff
@@ -59,6 +59,13 @@ namespace CalcEngine
 
     bool Number::IsZero() const
     {
-        return all_of(m_mantissa.begin(), m_mantissa.end(), [](auto &&i) { return i == 0; });
+        for (auto const& digit : Mantissa())
+        {
+            if (digit != 0)
+            {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Why did this `all_of` have to change?

Reply:
See: 49a291d9198045bdde2e16a7f7ef1b30f48c6103

Reply:
https://godbolt.org/z/eMGp-e Clang-5 has all_of capabilities, so most likely you're having problems with apple's clang or libcxx being bad....  it might have had something to do with the "not using c++17 standard" thing I mentioned in a previous comment

Reply:
@fwcd are you sure? I'm able to compile this with clang as ancient as 3.4 and GCC 4.9: https://godbolt.org/z/FflRkU

Reply:
@chuggafan That was indeed the issue, thanks.